frontend/public/components/utils/horizontal-nav.tsx

```diff
@@ -34,6 +33,7 @@ import {
   NavPage,
 } from '@console/dynamic-plugin-sdk/src/extensions/console-types';
 import { useExtensions, HorizontalNavTab, isHorizontalNavTab } from '@console/plugin-sdk/src';
+import { Helmet } from 'react-helmet-async';
```

Comment:
Nit - even though our ESLint rules are more lenient for `public` sources, I'd suggest to keep the import order as before.